### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Thomas Stäheli, Léonard Jouve and Ali Zoubir

### DISCUSSION THREAD

**Discussion by thomasstaheli** [heig-vd-dai-course/heig-vd-dai-course#530] [Show and tell]
Simulation of sales hotel of diffrents dofus items. For each client :
- he can sell items
- he can list the item he is selling
- he can list and buy other items from others clients
- each time a client sell an item, he receive money

Each client will login as a user. 
- Create a user
- Update / Delete user
- GET the list of user

**Comment by GeraudSilvestri** [comment]
Good for me

**Comment by thomasstaheli** [comment]
Bonsoir,

Voici le lien du répertoire : [HDV Dofus](https://github.com/thomasstaheli/DAI_Practical3_hdv_dofus)

Et voici le lien du commit hash : [7d3305f8a4a594fe724fe13c5cf3c34d44407363](https://github.com/thomasstaheli/DAI_Practical3_hdv_dofus/commit/7d3305f8a4a594fe724fe13c5cf3c34d44407363)

Bonne soirée et à demain !

**Comment by LeonardJouve** [comment]
Invitation à notre API postman https://app.getpostman.com/join-team?invite_code=1a6e59095b69c6bb858289582c8341327c3795b9d96c3e26e38a7a816db79fc1&target_code=65cdf77bce0a4e657227071420005b84
